src/views/DandisetLandingView/DandisetUnembargo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <v-card
+    outlined
+    class="mt-4 px-3"
+  >
+    <v-dialog
+      v-if="showWarningDialog"
+      v-model="showWarningDialog"
+      persistent
+      max-width="60vh"
+    >
+      <v-card class="pb-3">
+        <v-card-title class="text-h5 font-weight-light">
+          Release Dandiset
+        </v-card-title>
+        <v-divider class="my-3" />
+        <v-card-text>
+          <span>
+            This action will unembargo this dandiset. Note that this is a
+          </span>
+          <span class="font-weight-bold">
+            permanent
+          </span>
```

Comment:
Not a huge deal but I'm pretty sure you can nest this "inline" in the nonbold text currently in the `span` elements before and after this one.

Reply:
https://github.com/dandi/dandiarchive/pull/1021/commits/7bfc45f0c26446cbc2ab416bd2c4d8bf6052cf9b